src/main/java/uk/co/ramp/covid/simulation/output/DailyStats.java

```diff
@@ -212,4 +212,15 @@ public void determineRValues(Population p) {
         secInfections.set(rs.getSecInfections(day.get()));
         generationTime.set(rs.getMeanGenerationTime(day.get()));
     }
+
+    // Operates on the population *after* a run to determine those who died and were infected
+    // on the day this DailyStats tracks
+    public void determineFutureDeaths(Population population) {
```

Comment:
How about passing this function a list of all the people who are dead after a run, instead of passing the population?

Reply:
I think I prefer it this way for a documentation perspective - the conditional makes it clear that we are only counting people who are dead and who were infected on that day. If it's pre-filtered dead people that are passed in I think you lose this a bit. It's also less clear what the caller needs to do, since they are passing a List<Person> and could pass any arbitrary list of people by mistake (even if the parameter is called "deadPeople" it's easy to mistake this).